### ISSUE TITLE
Clarify page-progression-direction statement for fixed layout

### ISSUE THREAD

**Issue by dlazin** [mattgarrish/epub-revision#1883]
https://www.w3.org/TR/epub-rs-33/#sec-pkg-doc-spine says:

> Reading Systems MUST ignore the page progression direction defined in pre-paginated XHTML Content Documents. The page-progression-direction attribute defines the flow direction from one fixed-layout page to the next.

I don't understand what this is trying to say; can we clarify it? It's true that `page-progression-direction` defines the flow direction, but that's also true for reflowable EPUB. What's the special case for fixed layout here? That is, what specifically are reading systems supposed to be ignoring?

**Comment by dlazin** [comment]
Also worth noting that there is no normative statement *in the RS spec* that says "RSes must obey page-progression-direction." It's documented in the core spec (https://www.w3.org/TR/epub-33/#attrdef-spine-page-progression-direction), but it's not normative.

Put another way, handling `page-progression-direction="default"` is normative for RSes, but `ltr` and `rtl` are not.

**Comment by dlazin** [comment]
If you agree that `ltr` and `rtl` should be normative in the RS spec, please use this anchor and test references:

```
id="confreq-rs-spine-progression" data-tests="#confreq-rs-spine-progression_ltr,#confreq-rs-spine-progression_rtl"
```

I'm going to rename the corresponding tests (which @dauwhe already wrote) as above.

**Comment by mattgarrish** [comment]
> I don't understand what this is trying to say; can we clarify it?

FYI, this is essentially the same question as in https://github.com/w3c/epub-specs/issues/1483

**Comment by dlazin** [comment]
> FYI, this is essentially the same question as in #1483

Ah, excellent. I would dupe this issue, but that one doesn't note that the rtl/ltr values aren't currently normative, so I think this issue still has value. Feel free to dupe them if you prefer, just note that on the other side too.

**Comment by dlazin** [comment]
Update: as of the renaming in https://github.com/w3c/epub-tests/pull/73, this would be the attributes for the new normative statement:
```
id="confreq-rs-spine-progression" data-tests="#pkg-spine-progression_ltr,#pkg-spine-progression_rtl"
```

**Comment by dauwhe** [comment]
This is very closely related to #1483, which was closed by #1999
